modules/python/test/test_qrcode_detect.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+'''
+===============================================================================
+QR code detect and decode pipeline.
```

Comment:
Move test to the corresponding module:
https://github.com/opencv/opencv/tree/3.4/modules/objdetect/misc/python/test

Reply:
@alalek replaced